java/src/jmri/managers/AbstractManager.java

```diff
@@ -179,6 +179,11 @@ public void deleteBean(@Nonnull E bean, @Nonnull String property) throws Propert
     public void register(E s) {
         String systemName = s.getSystemName();
 
+        if (getBeanBySystemName(systemName) != null) {
```

Comment:
Is it an exception to register the same bean twice or to register two different beans with the same name?

I think those situations are slight different and should be handled accordingly.  I.e. registering the same bean twice should probably throw a warning, but it isn’t an exception while registering two different objects with the same name should throw an exception.

Reply:
I have changed the code to give only a warning if the same bean is registered twice.